_overviews/collections-2.13/conversion-between-option-and-collection.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: multipage-overview
+title: Conversion Between Option and Collection
+partof: collections-213
+overview-name: Collections
+
+num: 18
+previous-page: conversion-between-java-and-scala-collections
```

Comment:
Should we also set the `next-page` of the page `conversion-between-java-and-scala-collections`?